src/Section.tsx

```diff
@@ -73,19 +73,18 @@ const Section: React.FC<Props> = ({
   name,
   isOdd
 }) => {
-  console.log("IS ODD");
-  console.log(isOdd);
   const meetingDateTimes = [];
   const meetingDays: string[] = [];
   const meetingTimes: string[] = [];
   meetings?.forEach(m => {
     const dateTime = moment.utc(m.beginDate);
     meetingDateTimes.push(dateTime);
     meetingDays.push(dateTime.format("ddd"));
-    const endTime = dateTime.clone().add(m.duration, "minutes");
+    const endTime = dateTime.clone().add(m.minutesDuration, "minutes");
     meetingTimes.push(
       `${dateTime.format("h:mm A")}-${endTime.format("h:mm A")}`
     );
+    console.log("End Time is " + meetingTimes)
```

Comment:
Oops, you deleted my console logs and left your console logs. Lol

Reply:
lol. fixed